hvpy/api.py

```diff
@@ -1,13 +1,76 @@
 __all__ = ["fake_api_call"]
 
+from datetime import datetime
+from typing import Optional
+import requests
+from pydantic import BaseModel, Field
 
-def fake_api_call():
+
+class InputParameters(BaseModel):
+    date: datetime
+    sourceId: int
+    jpip: Optional[bool] = False
+    Json: Optional[bool] = Field(False, alias="json")
+
+
+def parse_response(response: requests.Response, output_parameters: str):
     """
     _summary_
 
+    Parameters
+    ----------
+    response: ?
+        _description_
+    output_parameters: ?
+        _description_
+
     Returns
     -------
     _type_
         _description_
     """
-    return {}
+    if output_parameters == "binary":
+        return response.content
+    elif output_parameters == "url":
+        return response.url
+    elif output_parameters == "json":
```

Comment:
Once you have the OutputType class, let's change these to OutputType.Raw, OutputType.String, and OutputType.Json